code/src/components/ParticipanceField.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const yesOrNoAnswers = ['Yes', 'No'];
+
+export const ParticipanceField = ({yesOrNo, setYesOrNo}) => {
+    
+    return (
+        <>
+            <label tabIndex="0" role="radiogroup" aria-required aria-label="Will you participate? Select Yes or No">Will you participate?</label>
+            {yesOrNoAnswers.map(answer => ( 
+                <label htmlFor="answer" key={answer} tabIndex="0">
+                    <input
+                        id="answer"
```

Comment:
I tested your site in Lighthouse and there are suggestions of 2 things you could do to improve accessibility: 

1. If you have the role="radiogroup" then you also need to have for example role="radio" for the children elements.
2. In line 13 the id should be unique. 

Try running Lighthouse again and you will see exactly what to do!